src/Bundle/Cli/Dumper/EmailDetailsDumper.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Sylius Sp. z o.o.
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Bundle\MailerBundle\Cli\Dumper;
+
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Style\SymfonyStyle;
+use Symfony\Contracts\Translation\TranslatorInterface;
+use Twig\Loader\LoaderInterface;
+
+final class EmailDetailsDumper implements EmailDetailDumperInterface
+{
+    public function __construct(
+        private readonly array $emails,
+        private readonly ?TranslatorInterface $translator,
+        private readonly LoaderInterface $templateLoader,
+    ) {
+    }
+
+    public function dump(InputInterface $input, OutputInterface $output, string $code): void
+    {
+        $email = $this->emails[$code];
```

Comment:
Case where email code is invalid and non existent in configured emails is not handled. As far as I can see it will output details snippet with invalid code, without subject and mark as `enabled: no`, while it doesn't exists at all.